src/general/Jukebox.cs

```diff
@@ -64,6 +66,15 @@ public string PlayingCategory
                 throw new ArgumentException("Playing category can't be set to null");
 
             GD.Print("Jukebox now playing from: ", value);
+            GD.Print("Active Music Contexts: ");
+
+            var contexts = string.Empty;
+
+            foreach (var ctx in activeContexts)
+                contexts += ctx + ", ";
```

Comment:
This could just print each one normally. String concatenation without a stringbuilder is not good C# usage.